tests/SpecTests/ChangeStreamsSpecTest.php

```diff
@@ -0,0 +1,240 @@
+<?php
+
+namespace MongoDB\Tests\SpecTests;
+
+use MongoDB\ChangeStream;
+use MongoDB\Client;
+use MongoDB\Driver\Exception\Exception;
+use ArrayIterator;
+use LogicException;
+use MultipleIterator;
+use stdClass;
+
+/**
+ * Change Streams spec tests.
+ *
+ * @see https://github.com/mongodb/specifications/tree/master/source/change-streams
+ */
+class ChangeStreamsSpecTest extends FunctionalTestCase
+{
+    /* These should all pass before the driver can be considered compatible with
+     * MongoDB 4.2. */
+    private static $incompleteTests = [
+        'change-streams: Test consecutive resume' => 'PHPLIB-416',
```

Comment:
There is actually no reason for this to fail. I believe the problem is that I'm using `iterator_to_array()` to capture the change stream result rather than the [documented iteration procedure](https://docs.mongodb.com/php-library/current/reference/method/MongoDBCollection-watch/#examples).

Will attempt to get this fixed and passing before we merge.

Reply:
Just to confirm, this was actually one of the two bugs cited below: [PHPLIB-442](https://jira.mongodb.org/browse/PHPLIB-442)